modules/itc/src/main/scala/lucuma/itc/legacy/encoders.scala

```diff
@@ -316,6 +318,14 @@ private given Decoder[SeriesDataType] = (c: HCursor) =>
     }
   }
 
+private given Decoder[ChartType] = (c: HCursor) =>
```

Comment:
`Enumerated`s already are `Decoder`s.

Reply:
Exactly that's why it lives in a specific `legacy` package to talk to the old ocs2 ITC